state/iaasmodel.go

```diff
@@ -26,10 +22,14 @@ func (m *Model) IAASModel() (*IAASModel, error) {
 	return &IAASModel{
 		Model: m,
 		mb:    m.st,
-		st:    m.st,
 	}, nil
 }
 
+// CloudRegion returns the name of the cloud region to which the model is deployed.
```

Comment:
it doesn't look like you removed the base Model method, so does this need to be here? I guess you added it and then abandoned the effort

Reply:
Yeah, it got too messy. And we'll be using region soon enough eg for federated k8s